docs/_includes/chart.html

```diff
@@ -0,0 +1,602 @@
+<div id="chart">
+    {% include section-header.html name="chart" version=page.versions.chart %}
+    <p>The chart module facilitates the layout of a chart, chart-legend, and any other applicable text (e.g. metric or subtitle text).</p>
+    <h3>Stacked chart with 3 segments</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div aria-label="Donut chart title" class="chart">
+                <div class="donut-graph chart__container">
+                    <svg width="190" height="190" viewBox="0 0 190 190" role="graphics-document">
```

Comment:
I don't think that `graphics-document` and `graphics-symbol` are ARIA roles. If not, these should be removed.

Reply:
I removed these.